infrastructure/lambda/template.yaml

```diff
@@ -273,12 +273,6 @@ Resources:
               Principal: '*'
               Resource:
                 - 'execute-api:/*'
-              Condition:
```

Comment:
This will cause breaking changes. 

If we look at the entire statement:

```
            - Action: 'execute-api:Invoke'
              Effect: Allow
              Principal: '*'
              Resource:
                - 'execute-api:/*'
            - Action: 'execute-api:Invoke'
              Effect: Deny
              Principal: '*'
              Resource:
                - 'execute-api:/*'
 ```

We are allowing all and also denying all. That no longer makes sense.